src/pages/spawn/spawnHost/EditSpawnHostModal.tsx

```diff
@@ -166,28 +170,37 @@ export const EditSpawnHostModal: React.FC<EditSpawnHostModalProps> = ({
             <InputLabel htmlFor="instanceTypeDropdown">
               Instance Types
             </InputLabel>
-            <Select
-              id="instanceTypeDropdown"
-              showSearch
-              style={{ width: 200 }}
-              placeholder="Select Instance Type"
-              onChange={(v) =>
-                dispatch({
-                  type: "editInstanceType",
-                  instanceType: v,
-                })
+            <Tooltip
+              title={
+                !canEditInstanceType
+                  ? "Cant change an instances type while it is running"
```

Comment:
this sentence is missing some apostrophes. can we change it to "Can't change and instance's type while it is running."

Reply:
or we can make the text more actionable: "Pause the host to adjust this field."

Reply:
Great suggestion! I had difficulty deciding on what the copy should say.